index.mjs

```diff
@@ -230,7 +230,7 @@ State.prototype['fantasy-land/ap'] = function ap(a) {
 //. A state monad parametrised by the type m of the state to carry.
 //.
 //. ```js
-//. > const Maybe = require ('sanctuary-maybe');
+//. > import Maybe from 'sanctuary-maybe';
 //. > const StateMaybe = StateT (Maybe);
```

Comment:
You should now be able to drop these semicolons.

Reply:
It seems like dropping the semicolons in between multiple lines starting with `>` causes the linter to indicate "missing semicolon".

Reply:
I see. Empty lines may be required. :thinking:

Reply:
That does the trick. I think I prefer the empty lines over the lines with semicolons for the sake of consistency, but it might be something to consider fixing.